python/grass/pygrass/vector/basic.py

```diff
@@ -133,9 +133,7 @@ def contains(self, point):
 
         """
         return bool(
-            libvect.Vect_point_in_box(
-                point.x, point.y, point.z if point.z else 0, self.c_bbox
-            )
+            libvect.Vect_point_in_box(point.x, point.y, point.z if point.z else 0, self.c_bbox)
```

Comment:
I’ll take a look and compare to my source branch. I have a big branch where I set up Pylint 3.x, and I fix issue after issue to reduce the diff of what I need to configure Pylint to ignore. Since having PRs merged is usually way too long here, I have them all in my branch, and then I cherry pick one commit to a new branch taken from main.

So the unrelated change might just be the fact that it isn’t in the same order as it was before being cherry picked, and I didn’t spot it when creating that branch.

I know that the combined changes are correct.

Reply:
done